### PR TITLE
Add visual-scripts package to automate build system

### PR DESCRIPTION
**Pull request by darthtrevino**
This package is inspired by the react-scripts model, where the build system is externalized from the target project. This eliminates a lot of duplication among our custom visual projects.

### CODE REVIEW

File: packages/visual-scripts/README.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+# @essex/powerbi-visual-scripts
+
+This package contains scripts for building PowerBI visuals using the Essex PowerBI build system.
+This build system is designed to emulate the high-level features of the PowerBI-visuals-tools, including the developer visual and bundling capabilities, but using webpack and more normative Typescript idioms (e.g. excluding namespaces, allowing imports).
+
```
Comment (stopyoukid) [inline packages/visual-scripts/README.md]: Some basic usage would be nice.

File: .travis.yml [hunk 2]
```diff
@@ -1,8 +1,7 @@
 language: node_js
 node_js:
-  - "node"
```
Comment (stopyoukid) [inline .travis.yml]: Why remove this, this will quickly tell us if the new version of node can compile it.

Comment (darthtrevino) [reply .travis.yml]: Oh, word, I removed 6 originally and was trying to put it back together

File: lerna.json [hunk 3]
```diff
@@ -3,10 +3,5 @@
   "packages": [
     "packages/*"
   ],
-  "version": "1.0.0",
-  "npmClient": "yarn",
-  "useWorkspaces": true,
```
Comment (stopyoukid) [inline lerna.json]: Is this line not necessary anymore? Did you run into problems with yarn workspaces?

File: package.json [hunk 4]
```diff
@@ -3,12 +3,12 @@
...
   "scripts": {
-    "clean": "del-cli \"./packages/*/lib\" \"!./packages/visual-styling/lib\"",
+    "clean": "lerna run clean --stream",
     "lint": "tslint -t verbose \"./packages/*/{src,spec}/**/*.ts\"",
-    "compile": "lerna run compile --stream",
+    "compile": "lerna run compile --concurrency=1 --stream",
     "docs": "lerna run docs --stream",
     "mocha": "mocha --require ts-node/register \"./packages/*/{src,spec,lib}/**/*.spec.ts\"",
-    "test": "lerna bootstrap && npm-run-all --sequential clean lint compile mocha docs",
+    "test": "npm-run-all bootstrap clean lint compile mocha docs",
```
Comment (stopyoukid) [inline package.json]: Wont this run into issues if `clean` ends up completing after `compile`?

Comment (darthtrevino) [reply package.json]: npm-run-all is sequential by default

File: packages/create-powerbi-visual/README.md [hunk 5]
```diff
@@ -0,0 +1,4 @@
+# Project Creator for Essex PowerBI Visuals
+
+## Usage
+`create-powerbi-visual <name>`
```
Comment (stopyoukid) [inline packages/create-powerbi-visual/README.md]: Is there any other configuration needed for this that would be nice in the docs?

Comment (darthtrevino) [reply packages/create-powerbi-visual/README.md]: Nope

File: packages/create-powerbi-visual/empty-visual/pbiviz.json [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  "apiVersion": "1.7.0",
+  "author": {
+    "name": "",
+    "email": ""
+  },
+  "assets": {
+    "icon": "assets/icon.png"
+  },
+  "externalJS": [
+    "node_modules/powerbi-visuals-utils-dataviewutils/lib/index.js"
```
Comment (stopyoukid) [inline packages/create-powerbi-visual/empty-visual/pbiviz.json]: :-1:

File: packages/create-powerbi-visual/README.md (hunk 5 above)
Comment (darthtrevino) [reply packages/create-powerbi-visual/README.md]: What may make sense is to update the new visual file's readme with some usage notes on how to use the new package.

File: packages/create-powerbi-visual/empty-visual/capabilities.json [hunk 7]
```diff
@@ -0,0 +1,84 @@
...
+                            "bind": {
+                                "to": "measure"
+                            }
+                        }
+                    ]
+                }
+            }
+        }
+    ]
+}
```
Comment (stopyoukid) [inline packages/create-powerbi-visual/empty-visual/capabilities.json]: You have a bunch of settings in here that aren't necessary for the visual to run

Comment (darthtrevino) [reply packages/create-powerbi-visual/empty-visual/capabilities.json]: It's the default stuff from `pbiviz new <derp>`

File: lerna.json (hunk 3 above)
Comment (darthtrevino) [reply lerna.json]: I was able to get past it, but I can't exactly tell you why it works now

**Pull request merged**